objects/StatTracker.af7ed7.ttslua

```diff
@@ -0,0 +1 @@
+require("playermat/statTracker")
```

Comment:
Since the unbundled code is less than 80 characters, ModManager won't extract it to a file.  It'll leave it in the LuaScript field in the .json, causing a change to that file.  It'll also leave this file orphaned.

Reply:
Understood and will be changed